backend/api/elite2Api.js

```diff
@@ -15,26 +15,26 @@ const elite2ApiFactory = client => {
 
   const del = (context, url, data) => client.del(context, url, data).then(processResponse(context))
 
-  const getAgencyDetails = (context, caseloadId) => get(context, `api/agencies/caseload/${caseloadId}`)
+  const getAgencyDetails = (context, caseloadId) => get(context, `/api/agencies/caseload/${caseloadId}`)
```

Comment:
ensure all apis start with `/` for consistency, superagent much more picky about it